OasisDEX/oasis-borrow
# Address discrepancy between After pills & Order Info

# [Investigate discrepancy between after pill amounts and order info](https://app.shortcut.com/oazo-apps/story/10592/investigate-discrepancy-between-after-pill-amounts-and-order-info)
  
## Changes 👷‍♀️
- Update decimal precision on after LTV in order info form
  
## How to test 🧪
- Go to /ethereum/ajna/multiply/ETH-USDC#setup
- Enter an ETH amt
- Confirm that after LTV marries up with after LTV in pill
